runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/FarmingContractManager.java

```diff
@@ -146,16 +146,19 @@ public boolean updateData(WorldPoint loc)
 		if (loc.getRegionID() == FARMING_GUILD_REGION_ID)
```

Comment:
Assign this to some `boolean inFarmingGuild` variable and reuse that in the infobox display check.